src/py42/clients/cases.py

```diff
@@ -0,0 +1,135 @@
+class CasesClient(object):
+    """A client to expose cases API.
+
+    `Rest documentation <https://default-cases.core-int.cloud.code42.com/swagger-ui.html#/Cases>`__
+    """
+
+    def __init__(self, cases_service, cases_file_event_service):
+        self._cases_service = cases_service
+        self._file_events = cases_file_event_service
+
+    @property
+    def file_events(self):
+        """A collection of methods for managing file events associated with a given case.
+
+        Returns :
+            :class:`py42.services.casesfileevents.FileEventsService`
+        """
+        return self._file_events
+
+    def create(
+        self, name, subject=None, assignee=None, description=None, findings=None
+    ):
+        """Creates a new case.
+
+        Args:
+            name (str): Name of the case.
+            subject (str, optional): User UID of a subject of a case.
+            assignee (str, optional): User UID of the assignee.
+            description (str, optional): Description of the case
+            findings (str, optional): Observations of the case.
+
+        Returns
+            :class:`py42.response.Py42Response`
+        """
+        return self._cases_service.create(
```

Comment:
Will need to make the change her as well:

https://github.com/code42/py42/pull/264/files#r552785803

Reply:
Changed.